test/e2e.test.js

```diff
@@ -13,6 +13,7 @@ test('demo app reports a vuln method when called', async (t) => {
       const beaconData = JSON.parse(requestBody);
       t.ok(beaconData.projectId, 'projectId present in beacon data');
       t.ok(beaconData.agentId, 'agentId present in beacon data');
+      t.ok(beaconData.systemInfo, 'systemInfo present in beacon data');
```

Comment:
I think maybe one of these should check that a non-error attribute is present in the `systemInfo`, if we're going to test it's there at all.